dti-website-redesign/src/components/Navbar.tsx

```diff
@@ -0,0 +1,45 @@
+import Link from 'next/link';
+import styles from './Navbar.module.css';
+
+// URLs will be the same as title, unless 'url' specified
+type NavbarItem = {
+  title: string;
+  url?: string;
+};
+
+const items: NavbarItem[] = ['team', 'products', 'courses', 'initiatives', 'sponsors', 'join'].map(
+  (title) => ({ title })
+);
+
+type Props = {
+  hidden?: boolean;
+};
+
+const Navbar = ({ hidden }: Props): JSX.Element =>
+  hidden ? (
+    <></>
+  ) : (
+    <div className={styles.navbarContainer}>
+      <>
+        <Link href="/">
+          {/* eslint-disable jsx-a11y/anchor-is-valid */}
+          <a className={styles.navLogo}>
+            <img className={styles.logo} src="/branding/wordmark.png" alt="wordmark" />
```

Comment:
I don't think this class is used in the module.css file?

Reply:
Yea I forgot that I moved it to the anchor tag. Removed now.